device/Microsoft.Azure.Devices.Client/ProductInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    class ProductInfo
+    {
+        public string Extra { get; set; } = "";
+
+        public override string ToString()
+        {
+            const string name = "Microsoft.Azure.Devices.Client";
```

Comment:
Nit: for new code we're following https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md 
`private const string Name = ...`

Reply:
Same with other places.

Reply:
Will do. In this case `name` is a variable in a method, so `private` doesn't make sense, but I get what you're saying.

Reply:
Ah, sorry... missed that.